### DISCUSSION TITLE
Particles support

### DISCUSSION THREAD

**Discussion by vujadin** [jnsmalm/pixi3d#13] [Ideas]
Would be nice to have particles support. 
Maybe using proton particles and creating pixi3D renderer for it? It has pixi (2D) support https://github.com/drawcall/Proton and there's three.js version https://github.com/drawcall/three.proton

**Comment by jnsmalm** [comment]
I agree, that would be nice. Do you mean 2D particles in 3D space?

**Comment by vujadin** [comment]
Well both 2D and 3D particles in 3D space I guess... 2D would be simple sprites (plane meshes always rotated toward camera) like this: https://drawcall.github.io/three.proton/engine/example/spriterender-g.html
3D would be any 3D mesh used as particle, like this: https://drawcall.github.io/three.proton/engine/example/meshrender-emitter.html or this: https://drawcall.github.io/three.proton/engine/example/customrender.html

**Reply by jnsmalm** [reply]
Latest version 0.9.4 now has support for instancing, which can be used for creating particles with meshes. See example at https://github.com/jnsmalm/pixi3d/blob/develop/examples/src/instancing.js
